src/widget.js

```diff
@@ -2,18 +2,23 @@
  * RemoteStorage connect widget
  * @constructor
  *
- * @param {object} remoteStorage - remoteStorage instance
- * @param {object} options - Widget options
- * @param {boolean} options.leaveOpen - Do not minimize widget when user clicks
- *                                      outside of it (default: false)
- * @param {number} options.autoCloseAfter - Time after which the widget closes
- *                                          automatically in ms (default: 1500)
- * @param {boolean} options.logging - Enable logging (default: false)
+ * @param {object}  remoteStorage          - remoteStorage instance
+ * @param {object}  options                - Widget options
+ * @param {boolean} options.leaveOpen      - Do not minimize widget when user clicks
+ *                                           outside of it (default: false)
+ * @param {number}  options.autoCloseAfter - Time after which the widget closes
+ *                                           automatically in ms (default: 1500)
+ * @param {boolean} options.skipInitial    - Don't show the initial connect hint,
+ *                                           but show sign-in screen directly instead
```

Comment:
This is missing the default value in the description.

Reply:
Thanks.